dspace-services/src/main/java/org/dspace/servicemanager/DSpaceServiceManager.java

```diff
@@ -426,10 +426,10 @@ private void registerBean(String name, Object service) {
                     service = (T) applicationContext.getBean(name, type);
                 } catch (BeansException e) {
                     // no luck, try the fall back option
-                    log.info(
+                    log.warn(
                         "Unable to locate bean by name or id={}."
                                 + " Will try to look up bean by type next."
-                                + " BeansException: {}", name, e.getMessage());
```

Comment:
Have you considered dropping `+ "BeansException: {}"` so that the Throwable argument triggers a stack trace?

Reply:
@cwilper : Yes, it looks like (to @mwoodiupui 's point), if this last line was just
```
+  name, e);
```
Then, you might get the full stacktrace in the logs (instead of just the string representation of the error).  I haven't dug deeply on this though, but it could be worth considering here. 

In any case, I think this PR is "good enough" as is....it's definitely an improvement over current behavior.

Reply:
Good catch @mwoodiupui , I missed that that arg was parameterized before and intended the exception details to be logged. Fixed.